docs/src/configuration/app/app-reference.md

```diff
@@ -605,6 +589,72 @@ crons:
 
 <!-- vale on -->
 
+### Cron job timing
+
+Minimum time between cron jobs being triggered:
+
+| Plan                | Time      |
+|-------------------- | --------- |
+| Professional        | 5 minutes |
+| Elite or Enterprise | 1 minute  |
+
+For each app container, only one cron job can run at a time.
+If a new job is triggered while another is running, the new job is paused until the other completes.
+To minimize conflicts, a random offset is applied to all triggers.
+The offset is a random number of seconds up to 5 minutes or the cron frequency, whichever is smaller.
+Crons are also paused while activities such as [backups](../../dedicated/overview/backups.md) are running.
+The crons are queued to run after the other activity finishes.
+
+To run cron jobs in a timezone other than UTC, set the [timezone property](#top-level-properties).
+
+### Paused crons
+
+Development environments are often used for a limited time and then abandoned.
+While it's useful for environments under active development to have scheduled tasks,
+unused environments don't need to run cron jobs.
+To minimize unnecessary resource use,
+crons are paused on Development environments with no deployments in 14 days.
```

Comment:
@CollierCZ Both Development and Staging environments (non-Production environment types).